src/pages/Admin/Admin.tsx

```diff
@@ -385,8 +389,11 @@ const VenueInfoComponent: React.FC<VenueInfoComponentProps> = ({
       </div>
       <VenueDeleteModal
         show={isDeleteModalVisible}
+        onCancel={hideDeleteModal}
         onHide={hideDeleteModal}
-        venue={venue}
+        onDelete={navigateToAdmin}
+        venueId={venue.id}
+        venueName={venue.name}
```

Comment:
Similar to `RunTabToolbar` props. Why the need to pass separate `id` and `name`?  `venue={venue}` serves the need for both

Reply:
Because we don't need the whole venue, you pass only the props you need. I will save 1 line of code.

Reply:
React's power is in the declarativity which makes it extremely readable.

This is much more readable and you immediately understand which fields from the venue are being used inside the component before you even open it and look at the logic inside.
```tsx
<VenueModal venueName={venue.name} venueDescription={venue.description} />
```


These two are voids of logic and you have no idea what's being passed down, you need to dive in and figure it out.
```tsx
<VenueModal venue={venue} />
```
```tsx
<VenueModal {...venue} />
```

Reply:
+ if anything changes in `venue` the component will re-render.

Reply:
> This is much more readable and you immediately understand which fields from the venue are being used inside the component before you even open it and look at the logic inside.

Why is the need to understand what an encapsulation does? 
Isn't the purpose of encapsulation to rid you of some concerns?

Let's imagine what `VenueInfoComponent` might say to a `VenueModal`:

1. > Here is my venue, now do what you need with it

vs

2. > Here are only 2 things, and I will not even tell you if they're from the venue, because I know what you're doing with those 2 fiel....

Just doesn't make sense 🤷 

Might be read differently if it was a `Modal`, instead of `VenueModal`, maybe it's only about a name change.

A component like that is expected to be generic, visual only without logic, so giving it only a `text` and/or just `onClick` would be more like the second approach.

Reply:
This component is specific for venue deletion and it's not an atom it's a molecule, it's concrete and makes sense to improve the readability and understand what props it requires.

Reply:
> Why is the need to understand what an encapsulation does?

Because that's one of the benefits of React and it's a standard/good practice. The declarativity is powerful and you immediately understand what props are passed down.

Reply:
I'm not debating about the existential questions of React.

I'm noting of the current PR code being between two equally valid opposing ends:

1. Have only `venueId` passed down and have their respective components determine if they need more by hook
2. Have only text and callback passed down and have the child components be unaware of venues at all.

Right now the components are half way neither way.